src/main/java/backend/resource/TurboIssue.java

```diff
@@ -438,13 +443,15 @@ public void addLabel(TurboLabel label) {
     public Optional<Integer> getMilestone() {
         return milestone;
     }
-
-    public void setMilestone(Integer milestone) {
-        this.milestone = Optional.of(milestone);
+    
+    public void setMilestoneById(Integer milestone) {
+        this.milestone = (milestone == null) ? Optional.empty() : Optional.of(milestone);
```

Comment:
The problem with passing null is that it doesn't inherently mean 'demilestone an issue'. Here we're using it to mean that, but it could also be a stray null that someone mistakenly passed in.

To be explicit, maybe we could have another method, `removeMilestone`? Then we can assume that this method will always have a non-null argument, representing a valid milestone.